app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_post_comments_counter
+  after_destroy :update_post_comments_counter
+
+  private
+
+  def update_post_comments_counter
+    post.update_comments_counter
+  end
```

Comment:
- [ ] Here, Also notice we need to increment the value of the `comments_counter` every time a new comment is added. Please make sure we are adding one more to the `comments_counter` property.